media/src/project/project-map-sidebar.tsx

```diff
@@ -152,40 +305,63 @@ export const ProjectMapSidebar = (
         addLayer();
     };
 
-    return (
-        <div id='project-map-sidebar'>
-            <h2>{title}</h2>
+    let panelState = 3;
```

Comment:
Abstracting the panel state here is a *great* pattern, and will help keep the code clean in the long term. I would suggest using constants here to clearly communicate intent rather than the raw numbers.

Reply:
Fixed